test/clj/rems/api/test_api.clj

```diff
@@ -70,18 +70,17 @@
                                                                :path "/api/c.*"}
                                                               {:method "get"
                                                                :path "/api/users/.*"}]})
+
     (testing "> api key without whitelist can access any path >"
-      (doseq [path ["/api/applications" "/api/my-applications" "/api/catalogue"]]
+      (doseq [path ["/api/applications" "/api/my-applications"]]
         (testing path
           (is (response-is-ok? (api-response :get path nil
                                              "45" "owner"))))))
     (testing "> api key with whitelist can access only given paths >"
       (doseq [path ["/api/applications" "/api/my-applications"]]
         (testing path
           (is (response-is-ok? (api-response :get path nil
-                                             "46" "owner")))))
-      (is (response-is-unauthorized? (api-response :get "/api/catalogue" nil
-                                                   "46" "owner"))))
```

Comment:
I think this is losing the unauthorized part of the test. Is it intentional that there is not such a test anymore?

Reply:
this could use "/api/catalogue-items" instead

Reply:
Yes, made it so